doc/frameworks/tensorflow/upgrade_from_legacy.rst

```diff
@@ -0,0 +1,254 @@
+######################################
+Upgrade from Legacy TensorFlow Support
+######################################
+
+With v2 of the SageMaker Python SDK, support for legacy SageMaker TensorFlow images has been deprecated.
```

Comment:
I'm open to suggestions. "v2.0.0" is more precise (since that'll be the release version), or could do something like "SageMaker Python SDK v2" or "SageMaker Python SDK 2"

Reply:
We can punt on this for now. Let's make a point to revisit it.